db/seeds.rb

```diff
@@ -5,14 +5,19 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
-image_files = Rails.root.join(Settings.IMAGE_PATH, '*.{jpg, tif, tiff, png}')
 
-Dir.glob(image_files)
-  .reject{ |f| f[/.*_tmp.*/] }
-  .map { |path| Image.from_file_path(path) }
-
-collection = Collection.find_or_create_by!(name: 'Default Collection')
-collection.images = Image.all
-collection.save!
-collection.create_composite_histogram
-collection.create_palette
+Dir.chdir("data/")
```

Comment:
This might be problematic. You may want to consider `Rails.root.join(Settings.IMAGE_PATH)` or something like that.

Reply:
I knew that this was probably bad Ruby code. I'll rework it.

Reply:
@mejackreed Is this (https://github.com/sul-cidr/histonets/pull/255/commits/c2343cf8eff580457c3b6dd52b4435d73a7608f9#diff-f7a25bc30a409fc77050703fc048ca8cR9) better or did you think I should completely avoid using `Dir.chdir()`?

Reply:
@csbailey5t Yeah I think that's better. I don't use it (perhaps just a stylistic thing though). I prefer not to have to think about where the current context of the directory is if possible. Potentially could lead to human mistakes? (I don't know.. if it works its probably fine)

Reply:
Also have you considered that we put the fast large storage in a different location on the server? Do we also need to have that configurable here too?

Reply:
I haven't.  I think that this is ultimately a stopgap measure to let us move forward with creating real collections and images from real data. Given that we need to figure out a real way of doing file IO for the whole app, with all the new data files we're writing and such, I think we'll definitely be revising this in the next work cycle.